docs/FAQs.md

```diff
@@ -3,8 +3,14 @@
 
 # Frequently Asked Questions
 
-## How is Hive different than the OpenShift installer?
+## How does Hive related to the OpenShift 4 installer (openshift-install)?
 
-* The OpenShift installer can be used to install/provision only one OpenShift cluster but Hive can be used for mutiple clusters.
-* Hive exposes an API to do provisioning/installation whereas the OpenShift installer requires a CLI.
-* The OpenShift installer does not manage the OpenShift clusters.
\ No newline at end of file
+Hive leverages the OpenShift 4 installer to perform actual cluster provisioning. We expose a similar API, but Hive has been built for managing a large number of clusters at scale, rather than just installing one. Hive offers a programatic API to control a number of day 2 configuration parameters, as well as a mechanism to manage arbitrary Kubernetes config in the resulting cluster.
+
+## Why doesn't Hive use Federation v2 for configuration management?
+
+During development Hive was initially being written to leverage Federation v2 for all day 2 configuration management in the cluster. Eventually however this effort had to be abandoned due to some conflicts in how Hive segregates cluster information by namespace, a lack of API stability in timeframes we needed, and some missing functionality due to our needs on applying cluster configuration, rather than application configuration. These issues were discussed with Federation developers and it's possible we may be able to resume this approach in the future.
```

Comment:
Have fixed small things and improved on the above paragraph. 

`However this effort had to be abandoned due to some conflicts in how Hive segregates cluster information by namespace, a lack of API stability in Federation V2 in the required time frame and some missing functionality around our need on applying cluster configurations rather than application configurations. These issues have been discussed with Federation developers. So there is a possibility that we might be able to resume Federation V2 approach in the future.
`